src/tests/mocks/accesscontrol_mocks.cairo

```diff
@@ -0,0 +1,209 @@
+#[starknet::contract]
+mod DualCaseAccessControlMock {
+    use openzeppelin::access::accesscontrol::AccessControl as accesscontrol_component;
+    use openzeppelin::access::accesscontrol::DEFAULT_ADMIN_ROLE;
+    use openzeppelin::introspection::src5::SRC5 as src5_component;
+    use starknet::ContractAddress;
+
+    component!(path: accesscontrol_component, storage: accesscontrol, event: AccessControlEvent);
+    component!(path: src5_component, storage: src5, event: SRC5Event);
+
+    #[abi(embed_v0)]
+    impl AccessControlImpl =
+        accesscontrol_component::AccessControlImpl<ContractState>;
+    #[abi(embed_v0)]
+    impl AccessControlCamelImpl =
+        accesscontrol_component::AccessControlCamelImpl<ContractState>;
+    #[abi(embed_v0)]
+    impl SRC5Impl = src5_component::SRC5Impl<ContractState>;
+
```

Comment:
Should we have `SRC5CamelImpl` here also?

Reply:
We are removing the support for SRC5Camel, not on this PRs, but we shouldn't add more dependencies IMO.

Reply:
I was thinking that, but it looks...wrong haha as long as we get rid rid of it before the next release